api/configuration/references.ts

```diff
@@ -28,7 +28,7 @@ export const SERVICES_CCD_DATA_STORE_API_PATH = 'services.ccd.dataApi'
 export const SERVICES_CCD_CASE_ASSIGNMENT_API_PATH = 'services.ccd.caseAssignmentApi'
 
 export const SERVICES_DOCUMENTS_API_PATH = 'services.documents.api'
-
+export const SERVICES_FEE_AND_PAY_API_PATH = 'services.feeAndPayApi'
```

Comment:
Do we need this? We not using feeAndPayApi

Reply:
i dont think we do. this was from the health check endpoint ticket merge. i did the rebase wrong yesterday, so its popped into this branch.

Reply:
please remove it. It will be an issue.

Reply:
its removed